nuodb/env-default

```diff
@@ -1,6 +1,12 @@
 #
-# default ENV VAR values
+# Default ENV VAR values. You must copy this to .env before running docker-compose.
+# DO NOT modify this file.
```

Comment:
We should clarify this to be: "Do not modify the 'env-default' file.

Reply:
Fair point.  I'll leave it with  you to merge this request and then make that change.